configs/platforms/osx-10.15-x86_64.rb

```diff
@@ -1,21 +1,3 @@
-platform 'osx-10.15-x86_64' do |plat|
-    plat.servicetype 'launchd'
-    plat.servicedir '/Library/LaunchDaemons'
-    plat.codename "catalina"
-    plat.provision_with 'export HOMEBREW_NO_EMOJI=true'
-    plat.provision_with 'export HOMEBREW_VERBOSE=true'
-    plat.provision_with 'sudo dscl . -create /Users/test'
-    plat.provision_with 'sudo dscl . -create /Users/test UserShell /bin/bash'
-    plat.provision_with 'sudo dscl . -create /Users/test UniqueID 1001'
-    plat.provision_with 'sudo dscl . -create /Users/test PrimaryGroupID 1000'
-    plat.provision_with 'sudo dscl . -create /Users/test NFSHomeDirectory /Users/test'
-    plat.provision_with 'sudo dscl . -passwd /Users/test password'
-    plat.provision_with 'sudo dscl . -merge /Groups/admin GroupMembership test'
-    plat.provision_with 'echo "test ALL=(ALL:ALL) NOPASSWD: ALL" > /etc/sudoers.d/username'
-    plat.provision_with 'mkdir -p /etc/homebrew'
-    plat.provision_with 'cd /etc/homebrew'
-    plat.provision_with 'su test -c \'echo | /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"\''
-    plat.provision_with 'sudo chown -R test:admin /Users/test/'
-    plat.vmpooler_template 'osx-1015-x86_64'
-    plat.output_dir File.join('apple', '10.15', 'puppet6', 'x86_64')
```

Comment:
We need to ship this packages to specific location so i guess this needs to stay.

Reply:
@jackie-kinsler @underscorgan i thought that this was used to define the package destination on agent-downloads. Eg. to ship OS X 10.15 packages under `agent-downloads/repos/apple/10.15/puppet6/x86_64/`

I'm not sure if removing this line will break any of the downstream automation but i'm ok to remove it if is not used anymore.

Reply:
I ran a couple of test builds and it looks like we should be okay getting rid of it.
This line: https://github.com/puppetlabs/vanagon/blob/main/lib/vanagon/platform.rb#L272
ensures that the output_directory is still meaningful by setting a default even if no output_dir is explicitly set.